examples/core-example-advanced/README.md

```diff
@@ -8,6 +8,11 @@ An advanced example project for `@tldraw/core`.
 
 To start this project:
 
-- run `yarn start:core` from the repository's root directory
-- open `localhost://5421` in your browser
-- OR run `yarn dev` from the `examples/core-example-advanced` directory
+1. Run `yarn` from the repository's root directory
+2. Run `yarn start:core` rom the repository's root directory
```

Comment:
A little typo in '*f*rom', same in the `tldraw-example` package

Reply:
fixed, good catch